test/kitchen/tasks/kitchen.py

```diff
@@ -216,12 +219,14 @@ def load_user_env(_, provider, varsfile):
     if os.path.exists(varsfile):
         with open(varsfile, "r") as f:
             vars = json.load(f)
-            for key, val in vars['global'].items():
-                if commentpattern.match(key):
-                    continue
-                env[key] = val
-            for key, val in vars[provider].items():
-                if commentpattern.match(key):
-                    continue
-                env[key] = val
+            if 'global' in vars:
```

Comment:
Instead of adding another level of nesting, would it make sense to do something like `for key, val in vars.get('global', {}).items()`? (And same below for `provider`)

Reply:
@bkabrda ready for another round of review when you are ready